### ISSUE TITLE
service terminated unexpectedly

### ISSUE THREAD

**Issue by henrist** [Cantara/Java-Auto-Update#15]
Observed: java-auto-updater process not running, windows logs showing unexpected error. The controlled application still running.

Should investigate what parameters available and set for automatically recover as a Windows service.

![started](https://cloud.githubusercontent.com/assets/703354/9862770/55902ad6-5b3b-11e5-814e-b5ba303944a0.png)
![dead](https://cloud.githubusercontent.com/assets/703354/9862772/5965e1c8-5b3b-11e5-8a91-3b4cad5f4a6d.png)
No related errors found in windows event log

Last events in jau.log:
```
13:01:09.393 [pool-1-thread-1] DEBUG n.c.j.s.client.ConfigServiceClient - Fetched ServiceConfig (length: 849).
13:01:09.393 [pool-1-thread-1] DEBUG n.c.j.s.client.ConfigServiceClient - Parsed serviceConfig (timestamp: 2015-08-11T10:13:12.141Z)
13:01:09.393 [pool-1-thread-1] DEBUG no.cantara.jau.Main - No updated config - checking if the process has stopped.
13:04:09.383 [pool-1-thread-1] DEBUG n.c.j.s.client.ConfigServiceClient - Fetched ServiceConfig (length: 849).
13:04:09.383 [pool-1-thread-1] DEBUG n.c.j.s.client.ConfigServiceClient - Parsed serviceConfig (timestamp: 2015-08-11T10:13:12.141Z)
13:04:09.383 [pool-1-thread-1] DEBUG no.cantara.jau.Main - No updated config - checking if the process has stopped.```

Analyzed graphs on Amazon EC2 without finding anything of interest

**Comment by henrist** [comment]
Hm. Acidentally I looked at the wrong logfile. It appears this happened on a connection error to ConfigService when updating config, causing an exception and termination of the service.

It should be resolved while working on #9 and when implementing this as a hystrixcommand with retries.

**Comment by jorunfa** [comment]
This should now be fixed in 5d565e637a241f0b4b272a3f0a045379d0eb115f and a3dbdd2a795a9d7839ba93869b83f104c3da94df.
